### ISSUE TITLE
[Localization] Missing Localizations

### ISSUE THREAD

**Issue by Vexatos** [SleepyTrousers/EnderIO#420]
Missing localizations:
- The "Insertion Filter" and "Extraction Filter" in the Item Conduit's GUI
- The names of the colours which you specify in the Item Conduit for Signal as well as the routing colour (the colour names are not localized in any conduit's GUI)
- The tooltip of the Contuit Facade always shows "Painted with:" untranslated
- The tooltip of the Fused Quartz Frame always shows "Painted with:" untranslated
- Capacitor Bank's tooltip always shows "Contains: x RF" untranslated
- Machine Chassi is not localized yet
- Basic Gear is not localized yet
- Basically, no tooltip is localized yet
- **Localization does _not_ update when switching the language, only changes on restart of the game**
